api.php

```diff
@@ -1008,6 +1008,61 @@ public function run($userID, $permissionIsExplicit) {
 	}
 }
 
+/**
+ * API entry game/getmessage
+ */
+class GetMessage extends ApiEntry {
+	public function __construct() {
+		parent::__construct('game/getmessage', 'GET', '', array('gameID','countryID','toCountryID','offset','limit'));
+	}
+	public function run($userID, $permissionIsExplicit) {
+
+		global $DB;
+		$args = $this->getArgs();
+
+		if ($args['gameID'] === null)
+			throw new RequestException('gameID is required.');
+
+		if ($args['countryID'] === null)
+			throw new RequestException('countryID is required.');
+
+		if (intval($args['limit']) > 25)
+			throw new RequestException('limit should not exceed 25');
```

Comment:
You should use the JSONresponse builder here to send back JSON data in the event of an exception. 

Also, this should account for invalid gameID/countryID args such as the use of characters. You should probably do a numeric check here.